scipy/stats/tests/test_multivariate.py

```diff
@@ -803,6 +805,18 @@ def test_mean_cov(self):
         ref = multivariate_normal.pdf(x, [1, 1, 1], cov_object)
         assert_equal(multivariate_normal.pdf(x, 1, cov=cov_object), ref)
 
+    def test_fit_error(self):
+        data = [1, 3]
+        error_msg = "`x` must be two-dimensional of shape (m, n)."
+        with pytest.raises(ValueError, match=re.escape(error_msg)):
+            multivariate_normal.fit(data)
+
+    def test_fit_correctness(self):
+        x = np.array([[-1., -1., -1.], [0., 0., 0.], [1., 1., 1.]])
+        mean_est, cov_est = multivariate_normal.fit(x)
+        assert_allclose(mean_est, np.zeros((3, )), atol=1e-15)
+        assert_allclose(cov_est, np.full((3, 3), 2/3), rtol=1e-15)
```

Comment:
For a stronger, more complete test that is a little easier to verify:

- Use random data
- Parameterize over `axis`
- Use `directional_stats` to compute the mean and `np.cov` (with `ddof=0`) to compute the covariance